aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move

```diff
@@ -4726,4 +5601,468 @@ module supra_framework::pbo_delegation_pool {
         assert_delegation(delegator1_address, pool_address, 101000000001, 0, 99999999999);
         unlock(delegator1, pool_address, 1000 * ONE_APT);
     }
+
+    #[test_only]
+    fun generate_multisig_account(owner: &signer, addition_owner: vector<address>, threshold: u64): address {
+        let owner_addr = aptos_std::signer::address_of(owner);
+        let multisig_addr = multisig_account::get_next_multisig_account_address(owner_addr);
+        multisig_account::create_with_owners(owner, addition_owner, threshold, vector[], vector[], 300);
+        multisig_addr
+    }
+
+    #[test(supra_framework = @supra_framework, validator = @0x123)]
+    #[expected_failure(abort_code = 327716, location = Self)]
+    /// if admin is option::none() calling to `replace_delegator` should fail
+    public entry fun test_repalce_delegration_without_multisig_failure(
+        supra_framework: &signer,
+        validator: &signer
+    ) acquires DelegationPoolOwnership, DelegationPool, GovernanceRecords, BeneficiaryForOperator, NextCommissionPercentage {
+        initialize_for_test(supra_framework);
+        account::create_account_for_test(signer::address_of(validator));
+        let delegator_address = vector[@0x010, @0x020];
+        let principle_stake = vector[100 * ONE_APT, 200 * ONE_APT];
+        let coin = stake::mint_coins(300 * ONE_APT);
+        let principle_lockup_time = 0;
+        let multisig = generate_multisig_account(validator, vector[@0x12134], 2);
+
+        initialize_test_validator(validator, 0, true, true, 0,
+            delegator_address,
+            principle_stake,
+            coin,
+            option::some(multisig),
```

Comment:
@nizam-supraoracles you are providing with `option::some(multisig)` but the heading of the test case says that this should fail if you pass `option::none()`, something is off here.